tests/test_regression.py

```
@@ -0,0 +1,160 @@
+import json
+import subprocess
+from pathlib import Path
+
+import pytest
+
+from langchain_opendataloader_pdf import OpenDataLoaderPDFLoader
+
+
+def java_available() -> bool:
+    try:
+        result = subprocess.run(["java", "-version"], capture_output=True, text=True)
```

Ruff (0.15.7)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)